"Developing a Grain" doc

1. 2.0 (.Net Standard based, right?) doc says to create a class project targeting .Net 4.6.1 - this is confusing
1. It says to install the nuget pkg `Microsoft.Orleans.OrleansCodeGenerator.Build` in to the class library, but merely doing this does not resolve the sample code properly (ie: `IGrainWithGuidKey`, `IGameGrain`, etc)
~~~
PlayerGrain.cs(10,37,10,54): error CS0246: The type or namespace name 'IGrainWithGuidKey' could not be found (are you missing a using directive or an assembly reference?)
PlayerGrain.cs(18,32,18,37): error CS0246: The type or namespace name 'Grain' could not be found (are you missing a using directive or an assembly reference?)
PlayerGrain.cs(12,14,12,24): error CS0246: The type or namespace name 'IGameGrain' could not be found (are you missing a using directive or an assembly reference?)
~~~

--- a/src/Documentation/grains/index.md
+++ b/src/Documentation/grains/index.md
@@ -5,11 +5,12 @@ title: Developing a Grain
 
 # Setup
 
-Before you write code to implement a grain class, create a new Class Library project targeting .NET 4.6.1 or higher in Visual Studio and add the `Microsoft.Orleans.CodeGenerator.MSBuild` NuGet package to it.
+Before you write code to implement a grain class, create a new Class Library project targeting .NET Standard (preferred) or .NET Framework 4.6.1 or higher (if you cannot use .NET Standard due to dependencies).
+Grain interfaces and grain classes can be defined in the same Class Library project or in two different projects for better separation of interfaces from implementation.
+In either case, the projects need to reference `Microsoft.Orleans.Core.Abstractions` and `Microsoft.Orleans.CodeGenerator.MSBuild` NuGet packages.
 
-``` powershell
-PM> Install-Package Microsoft.Orleans.CodeGenerator.MSBuild
-```
+For more through instructions, see the [Project Setup]( http://dotnet.github.io/orleans/Documentation/tutorials_and_samples/tutorial_1.html#project-setup) section of [Tutorial One – Orleans Basics]( http://dotnet.github.io/orleans/Documentation/tutorials_and_samples/tutorial_1.html).
+
 
 # Grain Interfaces and Classes
 

--- a/src/Documentation/grains/index.md
+++ b/src/Documentation/grains/index.md
@@ -9,7 +9,7 @@ Before you write code to implement a grain class, create a new Class Library pro
 Grain interfaces and grain classes can be defined in the same Class Library project or in two different projects for better separation of interfaces from implementation.
 In either case, the projects need to reference `Microsoft.Orleans.Core.Abstractions` and `Microsoft.Orleans.CodeGenerator.MSBuild` NuGet packages.
 
-For more through instructions, see the [Project Setup]( http://dotnet.github.io/orleans/Documentation/tutorials_and_samples/tutorial_1.html#project-setup) section of [Tutorial One – Orleans Basics]( http://dotnet.github.io/orleans/Documentation/tutorials_and_samples/tutorial_1.html).
+For more thorough instructions, see the [Project Setup](../tutorials_and_samples/tutorial_1.md#project-setup) section of [Tutorial One – Orleans Basics](../tutorials_and_samples/tutorial_1.md).
 
 
 # Grain Interfaces and Classes

